docs/config/shared-options.md

```diff
@@ -170,6 +170,13 @@ Enabling this setting causes vite to determine file identity by the original fil
 
 A nonce value placeholder that will be used when generating script / style tags. Setting this value will also generate a meta tag with nonce value.
 
+## html.chunkImportMap
+
+- **Type:** `boolean`
+- **Default:** `false`
+
+Whether to inject importmap for generated chunks. This importmap is used to optimize caching efficiency.
+
```

Comment:
I think we can set experimental without changing the name. 👍

Reply:
Nice, I need to use this feature as backend integration 👍 . It would be great if we could export `ImportMap` contents into a file like `import-maps.json`, and the consumers can name as the build.manifest feature (ref: https://vite.dev/config/build-options.html#build-manifest)